common-app-lib/src/cmr/common_app/api_docs.clj

```diff
@@ -151,23 +151,42 @@
   "Defines the footer of the generated documentation page."
   "</div></body></html>")
 
+(defn renderer
```

Comment:
Oh, and tested with both access-control-app and ingest-app: resulting files have the proper (new) timestamp and the generated docs files have all the expected content.